sdk/cosmos/azure-cosmos/src/test/java/com/azure/cosmos/implementation/ClientRetryPolicyTest.java

```diff
@@ -86,14 +86,6 @@ public void shouldRetryOnGatewayTimeout() throws Exception {
             .backOfTime(Duration.ofMillis(1000))
             .build());
 
-        //Data Plane Write
-        dsr = RxDocumentServiceRequest.createFromName(mockDiagnosticsClientContext(),
-            OperationType.Create, "/dbs/db/colls/col/docs/docId", ResourceType.Document);
-
-        clientRetryPolicy.onBeforeSendRequest(dsr);
-
```

Comment:
instead of removing the tests for write, I think we should validate the write operation will fail